runner/src/server/plugins/engine/models/SummaryViewModel.ts

```diff
@@ -511,7 +394,7 @@ export class SummaryViewModel {
   }
 
   get payApiKey() {
-    return this._payApiKey;
+    return this._payApiKey ?? "";
```

Comment:
I dont think i'd expect payApiKey to return an empty string if it didn't have a value.